### PR TITLE
remove excessive debug logging in ContinuableScopeManager

### PR DESCRIPTION
**Pull request by richardstartin**
The event being logged here *should* happen, unless something has gone wrong.

### CODE REVIEW

**Review by mcculls — APPROVED** [review]

File: dd-trace-core/src/main/java/datadog/trace/core/scopemanager/ContinuableScopeManager.java [hunk 1]
```diff
@@ -424,11 +424,8 @@ public AgentScope activate() {
       final boolean overrideAsyncPropagation = true;
       final boolean isAsyncPropagating = DEFAULT_ASYNC_PROPAGATING;
       if (used.compareAndSet(false, true)) {
-        final AgentScope scope =
-            scopeManager.handleSpan(
-                this, spanUnderScope, source, overrideAsyncPropagation, isAsyncPropagating);
-        log.debug("t_id={} -> activating continuation {}", spanUnderScope.getTraceId(), this);
```
Comment (tylerbenson) [inline dd-trace-core/src/main/java/datadog/trace/core/scopemanager/ContinuableScopeManager.java:430]: There's a similar log on line 520.

**Review by tylerbenson — APPROVED** [review]
Should we also consider reducing logging in `PendingTrace` (`registerContinuation`)?

File: dd-trace-core/src/main/java/datadog/trace/core/scopemanager/ContinuableScopeManager.java (hunk 1 above)
Comment (richardstartin) [reply dd-trace-core/src/main/java/datadog/trace/core/scopemanager/ContinuableScopeManager.java:430]: Thanks, missed that one

**Comment by richardstartin** [comment]
> Should we also consider reducing logging in PendingTrace (registerContinuation)?

@tylerbenson generally I think, when it's completely under our control like the scope manager, we should test more and log less, and keep very detailed diagnostic logging for environmentally sensitive things.

**Review by lpriima — APPROVED** [review]

**Pull request merged**